index.js

```diff
@@ -49,9 +49,12 @@ class ChartistGraph extends Component {
   }
 
   render() {
-    const className = this.props.className ? ' ' + this.props.className : ''
-    const style = this.props.style ? this.props.style : {};
-    return (<div className={'ct-chart' + className} ref='chart' style={style} />)
+    const { className, style, children, data, type } = this.props;
+    return (
+      <div className={`ct-chart ${className || ''}`} ref='chart' style={style}>
+         {children && cloneElement(children, { type, data })}
```

Comment:
are you able to just use `{...this.props.children}` or it's because you need to pass `type` and `data` down?